cobc/cobc.c

```diff
@@ -267,6 +268,8 @@ int			cb_flag_dump = 0;
 int			cobc_seen_stdin = 0;
 int			cb_unix_lf = 0;
 
+enum cb_ebcdic_table	cb_ebcdic_table = CB_EBCDIC_DEFAULT;
```

Comment:
please do not name an enum identical to a variable and consider to move the variable to cconv.c, external declaration to cconv.h

Reply:
So here we go for actual modularization contrary to almost everything else in `cobc`? Not that I'm not ok with that, just that it's a bit of a change ;-)

Reply:
That was one of the purposes of cconv* in general, so: yes.

This will also be an ongoing work for future changes (and in 4.x we already have split out our biggest player libcob/fileio.c); including the testsuite sources (the next point there is to split run_fileio.at and move out indexed tests to a separate file - because with 4.x where we can have multiple indexed handlers we actually want to run those tests with a single run under each indexed handler that is active.